source/Octodiff.Tests/Octodiff.Tests.csproj

```diff
@@ -37,8 +37,8 @@
     <Reference Include="WindowsBase" />
     <Reference Include="System" />
   </ItemGroup>
-  
-  <ItemGroup Condition="'$(TargetFramework)'=='netcoreapp2.0'">
+
+  <ItemGroup Condition="'$(TargetFramework)'=='net5.0'">
```

Comment:
Octodiff is used by `[Octopus.Client](https://github.com/OctopusDeploy/OctopusClients/blob/master/source/Octopus.Server.Client/Octopus.Server.Client.csproj#L48)`

Also `.netcoreapp3.1` is [supported longer](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) than net5.0